packages/nextjs/src/app-router/server/currentUser.ts

```diff
@@ -12,9 +12,7 @@ import { auth } from './auth';
  * - counts towards the [Backend API request rate limit](https://clerk.com/docs/backend-requests/resources/rate-limits#rate-limits).
  *
  * @example
- * ```tsx
- * // app/page.tsx
- *
+ * ```tsx {{ filename: 'app/page.tsx' }}
```

Comment:
I've changed my approach to indicate the filename to what we currently use in the docs. This won't show up in your IDE but we then can later use it in the docs.